src/elements/OcSidebarNavSubItem.vue

```diff
@@ -0,0 +1,56 @@
+<template>
+  <li :class="[active, 'uk-parent']">
+    <router-link :to="target">
```

Comment:
Had a quick go at wiring this up in Phoenix. We need to be able to push routes, with params. https://github.com/owncloud/phoenix/blob/master/src/components/Menu.vue#L107 Maybe we just pass in a route object including `name` and `params` elements?

Reply:
I guess we need that same for the breadcrumbs ....